maintenance/measure_tile_performance/measure_tile_performance.py

```diff
@@ -128,26 +153,31 @@ def main():
 
         i += 1
 
+    # print the filename on console
     if args.file:
         print("Filename =", filename)
         f.close()
         
     #Print for Console 
     print("RunTimes =", query_list)
-    minimum = min(query_list)
-    print("Min =", minimum)
-    maximum = max(query_list)
-    print("Max =", maximum)
+    #average run time
     sum_of_list = sum(query_list)
     length = len(query_list)
     average = round(sum_of_list/length, 2)
     print("Avg =", average)
+    #max run time
+    maximum = max(query_list)
+    print("Max =", maximum)
+    #min run time
+    minimum = min(query_list)
+    print("Min =", minimum)
```

Comment:
Why are you re-ordering the output?

Reply:
This is being re-ordered because in https://apps.itsm.gov.bc.ca/confluence/display/ANALYTICS/Measure+Looker+Tile+Performance the order of columns in all tables. When we type this information from cron email, it is easier to look and type if both are in same order. In the past i had some typos while inputting these measurements in the document.

Reply:
I see. Okay, I guess that's fair then.